From f8f65d332a26103ddc6458db3e9591d7abe6826d Mon Sep 17 00:00:00 2001
From: Leon Romanovsky <leonro@nvidia.com>
Date: Wed, 28 Oct 2020 16:41:12 +0200
Subject: [PATCH] redhat: Fix wrong installation path of ib_acm

+ bin/ib_acme -D . -O
/var/tmp/rpm-tmp.4e99gS: line 54: bin/ib_acme: No such file or directory

Signed-off-by: Leon Romanovsky <leonro@nvidia.com>
---
 redhat/rdma-core.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/redhat/rdma-core.spec b/redhat/rdma-core.spec
index 817dc208..c468a493 100644
--- a/redhat/rdma-core.spec
+++ b/redhat/rdma-core.spec
@@ -346,7 +346,7 @@ rm -f %{buildroot}%{_sysconfdir}/rdma/modules/rdma.conf
 install -D -m0644 redhat/rdma.conf %{buildroot}%{_sysconfdir}/rdma/modules/rdma.conf
 
 # ibacm
-bin/ib_acme -D . -O
+%{buildroot}%{_bindir}/ib_acme -D . -O
 install -D -m0644 ibacm_opts.cfg %{buildroot}%{_sysconfdir}/rdma/
 
 # Delete the package's init.d scripts
-- 
2.28.0

